app/controllers/host_shelters/host_shelters_controller.rb

```diff
@@ -0,0 +1,10 @@
+class HostShelters::HostSheltersController < ApplicationController
+  helper_method :auth_user_check?
+
+  def auth_user_check?
+    unless current_user
+       flash[:errors] = "Login Please"
+       return false
```

Comment:
@swdonovan I reread your comments and I understand now what you meant.  The method works, I would just add a redirect to the login or wherever you feel it's appropriate to go with the flash message.  The flash message isn't showing I think that just needs to be added to the views.

Reply:
@erose357 1) sorry this im just getting back to you on this, the day became interesting/chaotic 45 minutes after i started this conversation so i apologize.
2)Thank you for the redirect advice this "auth_user_check?."  That made perfect sense and is further proof of why +1 additional brain, +2 eyes and pomodoro breaks are important.